### DISCUSSION TITLE
TTS model deployment on Nvidia Triton server tutorial is required.

### DISCUSSION THREAD

**Discussion by ghost** [NVIDIA-NeMo/NeMo#4890] [Q&A]
I was looking over the web and I couldn't find any tutorial or notebook which deploys NeMo TTS models like fastpitch and univnet on triton server. Both of these models can be converted in Onnx format, It would be a great help for this community if someone can share the tutorial or a notebook.

**Comment by XuesongYang** [comment]
@blisc do you have any pointer to the guidance?
